### PR TITLE
Expanding Flask example with additonal config details

### PR DESCRIPTION
**Pull request by tremlab**

### CODE REVIEW

File: example/flask/README.md [hunk 1]
```diff
@@ -2,61 +2,32 @@
...
-pip install -r requirements.txt
-```
+Try this out with [your own Bugsnag account](https://app.bugsnag.com/user/new)! You'll be able to see how the errors are reported in the dashboard, how breadcrumbs are left, how errors are grouped and how they relate to the original source.
 
-Before testing it, open up the `server.py`
+To get set up, follow the instructions below. Don't forget to replace the placeholder API token with your own!
+
+1. Clone the repo and `cd` into this directory:
+    ```sh
+    git clone git@github.com:bugsnag/bugsnag-python.git
```
[the remark this answers is not in the document]
Comment (tremlab) [reply example/flask/README.md]: excellent point! I will keep this in mind for my PRs.  

Thanks!

File: example/flask/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <h3>This page is a basic example of how to include bugsnag-python in a flask app.
+    </h3>
+  </div>
+
+    <br>
+  <h3>Click below to send errors to your Bugsnag dashboard.</h3>
+  <br>
+  <br>
+    <div id="python">
+        <a href="/crashdict">
```
Comment (Cawllec) [inline example/flask/templates/index.html]: We should have some styling on these `a` elements could be added to stop them from being underlined.
Something like `style="text-decoration:none"`.

**Comment by tremlab** [comment]
made updates per comments, confirmed it works for both python 2 & 3.

**Comment by Cawllec** [comment]
I've fixed the underlining and merged this manually .

**Pull request closed without merging**